packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -642,7 +642,7 @@ void main() {
     expect(find.byType(EditableText), paints
       ..rrect(
         rrect: RRect.fromRectAndRadius(
-          Rect.fromLTRB(191.3333282470703, 2.0833332538604736, 193.3333282470703, 14.083333015441895),
+          Rect.fromLTRB(191.3333282470703, 0.0833332538604736, 193.3333282470703, 16.083333015441895),
```

Comment:
How were we able to accidentally change the cursor size over time with these tests in place?  Or were they created after the cursor was already the wrong size?

Reply:
I think it is that it never truly correct in the first place. The existing versions work *okay* for alphabetic scripts, but ideographic scripts have always had the problem of strange/too short heights. Upon closer inspection and comparison with the android carets, it seems that this taller version is both better for ideographic scripts as well as more faithful for alphabetic scripts. Either way, what we do here is only an emulation of the actual Android/iOS carets.

Reply:
Maybe consider adding a test case for ideographic characters then, if you think it would help catch more cursor regressions.  It looks like all of these use alphabetic characters.

Reply:
Test added!

Reply:
Looks great, thanks!